src/style.css

```diff
@@ -0,0 +1,86 @@
+body {
+  margin: 0;
+}
+
+.container {
+  max-width: 1200px;
```

Comment:
that's not necessary

Reply:
I know, but somewhy cannot solve the test without it.